src/TLuaInterpreterMapper.cpp

```diff
@@ -1560,6 +1560,36 @@ int TLuaInterpreter::getExitStubs1(lua_State* L)
     return 1;
 }
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#getExitStubsNames
+int TLuaInterpreter::getExitStubsNames(lua_State* L)
+{
+    const QStringList stubmap = { "north", "northeast", "northwest", "east", "west",
+                                  "south", "southeast", "southwest", "up", "down", "in",
+                                  "out", "other"
+    };
+
+    const Host& host = getHostFromLua(L);
+    if (!host.mpMap || !host.mpMap->mpRoomDB) {
+        return warnArgumentValue(L, __func__, "no map present or loaded");
+    }
+
+    const int roomId = getVerifiedInt(L, __func__, 1, "roomID");
+
+    // Previously threw a Lua error on non-existent room!
```

Comment:
This comment probably came from somewhere else, but neither here or there does it add any value :sweat_smile:

Reply:
lol, noted